xml/System.Data.Common/DbCommand.xml

```diff
@@ -1472,7 +1472,7 @@
       </Docs>
     </Member>
     <Member MemberName="PrepareAsync">
-      <MemberSignature Language="C#" Value="public virtual System.Threading.Tasks.Task PrepareAsync (System.Threading.CancellationToken cancellationToken = null);" />
+      <MemberSignature Language="C#" Value="public virtual System.Threading.Tasks.Task PrepareAsync (System.Threading.CancellationToken cancellationToken = CancellationToken.None);" />
```

Comment:
Oh sorry if I made a mistake - this is my first time contributing API docs here. I was following @divega's lead [in these PRs](https://github.com/dotnet/dotnet-api-docs/pulls?utf8=%E2%9C%93&q=is%3Apr+author%3Adivega) and was under the impression these XMLs had to be modified manually.

Reply:
Oh I see - I'm guessing you meant changes in the signature specifically, as opposed to the descriptions.

In that case there does seem to be an issue: CancellationToken is a struct, so it cannot have a default value of null. I'm not sure how this could be handled without manual intervention.

Reply:
OK, I'll revert this part tomorrow.

Reply:
@rpetrusha thanks!